### PR TITLE
login screen UI implemented

### PR DESCRIPTION
**Pull request by DBashir**

### CODE REVIEW

File: app/src/main/java/com/example/badgermecohort1/navigation/Login.kt [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            }
+        }
+        Row(modifier = Modifier.padding(bottom = 40.dp, start = 15.dp, end = 15.dp)) {
+            Button(
+                onClick = {},
+                modifier = Modifier
+                    .clip(RoundedCornerShape(15.dp, 15.dp, 15.dp, 15.dp))
+                    .fillMaxWidth()
+            ){
+                Text("Sign in with Google")
```
Comment (TimboTambo) [inline app/src/main/java/com/example/badgermecohort1/navigation/Login.kt:112]: Should this text be localised?

File: app/src/main/java/com/example/badgermecohort1/navigation/Login.kt [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+            contentDescription = "rb title",
+            modifier = Modifier.padding(Dp(50.0F))
+        )
+        Column(
+            horizontalAlignment = Alignment.CenterHorizontally,
+            verticalArrangement = Arrangement.Center,
+            modifier = Modifier.weight(1f, true)
+        ) {
+            Row() {
+                Image(painter = painterResource(R.drawable.square),
```
Comment (TimboTambo) [inline app/src/main/java/com/example/badgermecohort1/navigation/Login.kt:49]: Guess we could make these repeated images a component but that can be a future optimisation.

File: app/src/main/java/com/example/badgermecohort1/navigation/Navigation.kt [hunk 3]
```diff
@@ -18,10 +19,7 @@ fun navigation() {
             SplashScreenAnimate(navController)
         }
         composable("main_screen") {
```
Comment (TimboTambo) [inline app/src/main/java/com/example/badgermecohort1/navigation/Navigation.kt:21]: We could rename this as "login_screen" in the nav now to make it clearer?

File: app/src/main/java/com/example/badgermecohort1/navigation/Login.kt [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+                    contentDescription = "square1",
```
Comment (TimboTambo) [inline app/src/main/java/com/example/badgermecohort1/navigation/Login.kt:50]: Might be good to name these images based on the category that they represent for when we come to use them elsewhere in the app, but again that could always be done later.

**Review by TimboTambo — APPROVED** [review]

File: app/src/main/java/com/example/badgermecohort1/navigation/Login.kt (hunk 4 above)
Comment (DBashir) [reply app/src/main/java/com/example/badgermecohort1/navigation/Login.kt:50]: added todo in code

File: app/src/main/java/com/example/badgermecohort1/navigation/Navigation.kt (hunk 3 above)
Comment (DBashir) [reply app/src/main/java/com/example/badgermecohort1/navigation/Navigation.kt:21]: renamed

File: app/src/main/java/com/example/badgermecohort1/navigation/Login.kt (hunk 2 above)
Comment (DBashir) [reply app/src/main/java/com/example/badgermecohort1/navigation/Login.kt:49]: added todo in code

**Pull request merged**